src/status_im/contexts/profile/push_notifications/local/events.cljs

```diff
@@ -32,7 +43,9 @@
    bodyType))
 
 (rf/defn process
-  [cofx event]
-  (if platform/ios?
-    {:effects/push-notifications-local-present-ios (create-notification nil event)}
-    {:effects/push-notifications-local-present-android (create-notification cofx event)}))
+  [cofx {:keys [bodyType] :as event}]
+  (if (push-notification-types bodyType)
```

Comment:
make sure that you captured all the types in the list please (looks like, but double check).

Reply:
Checked, Thanks